_integration-schemas/slack/v1/threads.md

```diff
@@ -0,0 +1,78 @@
+---
+tap: "slack"
+version: "1"
+key: ""
+
+name: "threads"
+doc-link: "https://api.slack.com/docs/conversations-api"
+singer-schema: "https://github.com/singer-io/tap-slack/blob/master/tap_slack/schemas/threads.json"
+description: |
+  The `{{ table.name }}` table contains information about conversation threads in your {{ integration.display_name }} team. This table will do a Full Table Replication for each parent message that is added.
```

Comment:
> This table will do a Full Table Replication for each parent message that is added.

While technically correct, I think the phrasing here is a little awkward. Also, is it the **table** that replicates in full, or the **message**? The way I interpret this is that each time a message in a thread is updated, the entire thread and all the messages it contains will be replicated in full. Is that correct?